### PR TITLE
kristindawn 01addition

### PR DESCRIPTION
**Pull request by kristindawn**
This is a first go at refreshing the introductory episode. I created it as 01 but it would (in theory) replace 02. Take a look, see what you think. I think it could use another activity but I wanted a check-in before doing any more work on it.

### CODE REVIEW

File: _episodes/01-intro-to-library-carpentry.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+
+### How?
+
+Libraries have been collecting data for centuries in the form of catalog records, but we also collect data through other methods. Some of those methods are automatically built into our systems, like circulation data, and others require new datasets and data collection instruments to be created, like a survey of community members. The method you use to collect your data will influence how it can be used and what kinds of conclusions you can draw from it. Keeping track of how you collected the data will help you repeat the process in the future so that you can compare results over time.   
+
+> ## Documenting your data
+> Whether you create a text README file, a data dictionary, a codebook, or a project notebook, it is important
+> to document your data. This includes things like:
+> - where your came from
```
Comment (PhilReedData) [inline _episodes/01-intro-to-library-carpentry.md:81]: + > - where your data came from

File: _episodes/01-intro-to-library-carpentry.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+No matter where you are storing your data you should consider any security concerns, particularly when dealing with any data that might lead to the identification of your community members. If it is essential that you collect and store this information, check in with your local IT group to learn about which systems at your institution are safe for storing sensitive information.
+
+> ### Data about human subjects
+> 
+> Any time we collect and use data about other humans we have to think carefully about why, how, and where we are storing it.
+> Like so many systems that we use every day, our library and learning management systems are collecting increasing amounts of data about the people who use them, and we
+> need to respect their privacy. If you plan to use any data about your communities in presentations or publications it is a good idea to go through the process 
+> of filling out a request with your local Institutional Review Board (IRB) and going through the ethics training program. It is a great way to learn more about human
+> subjects research.
```
Comment (PhilReedData) [inline _episodes/01-intro-to-library-carpentry.md:103]: + > subjects research. You should also check what legal requirements are relevant in your country (for example, GDPR in the European Union).

File: _episodes/01-intro-to-library-carpentry.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+We want to explore the circulation statistics of our physical collection outside of our integrated library system (ILS). 
+1. We export the data from the ILS as a CSV (comma separated values) file, giving it the name **2021-05-24_ilsExport_raw_v01.csv** and save it to a folder called **2021_rawData**. 
+2. We know that the raw data has errors accumulated over decades of changing cataloging practice, so we open the CSV file in **OpenRefine** so we can use the faceting tools to find possible inconsistencies. 
+3. Once we are done, we export the data as a CSV to a new folder called **2021_processedData** using the name **2021-05-27_ilsExport_proc_v01.csv**. 
+4. We also export the JSON file of each step we used to clean our data in OpenRefine so we know what we did and can do it again. (more in the [OpenRefine Lesson](https://librarycarpentry.org/lc-open-refine/)).
```
Comment (PhilReedData) [inline _episodes/01-intro-to-library-carpentry.md:117]: ... "More" with a capital M

**Comment by kristindawn** [comment]
Hi! I just want to circle back to this. Is it worth continuing to build out this episode? I might use it next week in a session by making a copy so any additional thoughts would be great.

**Comment by libcce** [comment]
@kristindawn I think you should test out your concept/approach. The feedback from the session will be helpful! And then you can come back here again. You shouldn't hold yourself back :)

**Comment by morskyjezek** [comment]
Hi all, this PR has been hanging out here for about 2 years. Has any follow up happened? I like the structure of the introduction and overview that you proposed, @kristindawn - was there any feedback from lessons or others as suggested by @libcce ? 

I think the overview could use a refresh, and this would be great content to do that. On the other hand, I am not sure about removing the current page altogether. Are there any further thoughts on this?

**Comment by morskyjezek** [comment]
These are helpful suggestions, but I am going to close this PR since we are working through the transition to the new Carpentry workbench platform. These suggestions have been under discussion for about 2 years, but I don't see the follow up that was requested. I would suggest that these changes be converted into an open issue and discussed by the Library Carpentry Curriculum committee.

**Pull request closed without merging**